litecommands-unit/src/dev/rollczi/litecommands/unit/AssertSuggest.java

```diff
@@ -51,6 +54,14 @@ public AssertSuggest assertSuggest(String... suggestions) {
         return assertSuggest(Arrays.asList(suggestions));
     }
 
+    public AssertSuggest assertSuggest(String[] suggestions, String... tooltips) {
```

Comment:
If we want to assert suggestions with tooltip, then I think we should create method with varargs Suggestion... parameter